### ISSUE TITLE
Steam Deck - D3D11 renderer does not work

### ISSUE THREAD

**Issue by Mutant22** [kirides/GD3D11#98]
**Describe the bug**
GD3D11 renderer does not work on steam deck. With same steps I could run it on desktop machine, but not on steam deck.

**To Reproduce**
1. I have installed steam version of Gothic 2 Gold
2. Set compatibility tools to Proton 7.0-1
3. Installed libraries through protontricks - d3dcompiler_43, d3dcompiler_47, d3dx11_43, d3dx9_43, vcrun2008, vcrun2010, vcrun2012, vcrun2013
4. Extracted d3d11 renderer v17.8-dev4-ci to gothic system folder, and removed d3dcompiler_47.dll from it
5. Added ddraw (native, builtin) to wine overrides

With v17.8-dev4-ci version I got warning on start "GD3D11 Renderer couldn't be loaded. Access Denied(0x7e)" and game started without directx 11 renderer.

With older version X17.8-dev1, game foze on start. Log.txt was created, so I adding it here.
[Log.txt](https://github.com/kirides/GD3D11/files/8336068/Log.txt)

**Screenshots**
![Screenshot_20220323_204053](https://user-images.githubusercontent.com/19385267/159782677-c0461eec-c720-476f-a9bd-2b95c5bb58b2.png)

**Please complete the following information):**
 - Graphics card: AMD RADV VANGOGH
 - Gothic version: Steam version of Gothic 2 Gold

**Comment by kirides** [comment]
Did you try to roughly follow this https://github.com/kirides/GD3D11/issues/14 ?

**Comment by Mutant22** [comment]
Yes I read that bugreport, thats why I removed that d3dcompiler dll, and installed overrides through protontricks. I can try using lutris, or clear wine.

**Comment by Mutant22** [comment]
Creating 32 bit wineprefix through "Bottles" and using Proton-6.21-GE-2 seems to work.
Enabled DXVK in preferences, added ddraw override and installed dependencies (vcredist2008, vcredist2010, d3dcompiler_47).
I then manually copied steam game installation into prefix, and now it started successfully.

**Comment by Mutant22** [comment]
Alright, this way it only worked in desktop mode, and not in game mode.
And mouse had problem with moving camera, it was like mouse stuck to the edge of screen.

I will need to investigate it further.

**Comment by Mutant22** [comment]
EDIT: It works now. I found some strange bug with extracted GD3D11 folder. 
Renderer fails to load, but when I create tmp folder,copy content of GD3D11 to tmp folder. Remove original GD3D11 folder, and then renaming tmp folder to original name it loads now. I am not sure wheter some attributes in zip archive, are set on that folder. But it would be some Linux/wine shenengian. You can close this.

**Comment by Mutant22** [comment]
Here is bugvideo: https://mega.nz/file/a5gm3CLQ#R9onIvwt3nW3p-HtQoiF89yfD9YZ8qSaTp_LGM6CsfM

**Comment by Mutant22** [comment]
It was not problem with GD3D11 itself. Maybe some problem with package extraction, or linux itself.

**Comment by emosGambler** [comment]
Had the same error after most recent release. I had manually swap the mod files to previous release and it worked.

**Comment by fhd** [comment]
Sorry for the confusion, turns out I was using this one: https://github.com/SaiyansKing/GD3D11

Apparently the latest release broke things on Steam Deck (access denied), but luckily someone else had also uploaded an (as of yet not updated, hence working) version of it to Steam Workshop.
